[TASK] Small readme updates

erikhansen opened the pull request:
Added links to a couple of helpful resources regarding email client support for CSS

Added instructions for installing PHPUnit and running tests

oliverklee commented on README.md:
```diff
@@ -143,6 +146,9 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, [install PHPUnit](https://phpunit.de/getting-started.html) and then run this
```
Actually, you should install the development-related composer packages using composer:

composer install

(omitting the --no-dev option then also installs the development packages).

PHPUnit will then be available as vendor/bin/phpunit .

oliverklee commented on README.md:
```diff
@@ -143,6 +146,9 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, [install PHPUnit](https://phpunit.de/getting-started.html) and then run this
+      command in the root of the GIT repo: 
+      `phpunit --bootstrap Classes/Emogrifier.php  Tests/Unit/EmogrifierTest.php`
```
We also don't need the bootstrap parameter here.

oliverklee commented:
I've fixed the PHP_CodeSniffer warnings on Travis. Could you please rebase and re-push? Thanks!

erikhansen commented:
@oliverklee - I've rebased the commit and updated the document to cover the proper method of installing phpunit. Thanks for informing me of that. I guess this experience demonstrates the value of having such instructions in the readme.

oliverklee commented on README.md:
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
+     
+      and then run this command:
+      
+      `vendor/bin/phpunit Tests/Unit/EmogrifierTest.php`
```
Actually, providing just the Tests/ directory as a parameter should suffice.

oliverklee commented on README.md:
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
```
Could you please adapt the indentation to match the indentation of the previous lines?

oliverklee commented on README.md:
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
+     
+      `composer install` 
```
For code lines, if you indent with four spaces, you won't need the backticks for marking the text as code.

Please see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code for a Markdown cheat sheet.

oliverklee commented on README.md:
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
+      
+      `vendor/bin/phpunit Tests/Unit/EmogrifierTest.php`
+      
+      If you want to uninstall the development packages that were installed when you ran `composer install`, you can 
```
Could you please change the (new) Markdown lines to be 80 characters or less? This will avoid wrapping on GitHub.

oliverklee commented:
For the commit message:

* Could you please wrap the lines in the description at 72 characters?
* Could you please a bullet lists (using asterisks) for the lines in the description that are no paragraphs?

For more information on good commit messages, please see http://chris.beams.io/posts/git-commit/ .

erikhansen commented:
@oliverklee I've made the requested changes and pushed the rebased commit.

oliverklee commented:
Thanks!
